bin/realizeCascades.cpp

```diff
@@ -55,7 +55,7 @@ void print_usage (FILE* stream, int exit_code)
            "  -n, --numgen        <number>       number of traces to generate \n"
            "  -o, --outfile       <filename>     name the output file \n"
            "  -s, --silent                       silent, no standard out \n"
-           "  -v, --verbose       <level>        Print verbose messages at level <level>\n"
+           "  -v, --verbose       <level>        Print verbose messages at level <level>. Currently must use `--verbose=<level>` or `-v<level>` - no spaces.\n"
```

Comment:
do we need a new line or something to make this format properly? IDK I didn't check.

Reply:
Here's the help output as it is:

```
gerudo7@Teldrassil ~/g/nrCascadeSim (bugfix/documentation-issues)> realizeCascades --help                    (root_env)
Usage:  (null) options [ inputfile(s) ]
  -d, --seed          <integer>      seed for random numbers
  -h, --help                         print usage
  -n, --numgen        <number>       number of traces to generate
  -o, --outfile       <filename>     name the output file
  -s, --silent                       silent, no standard out
  -v, --verbose       <level>        Print verbose messages at level <level>. Currently must use `--verbose=<level>` or `-v<level>` - no spaces.
  -V, --version                      print version and exit
  -l, --log           <filename>     Log additional output to the specified file. If this option is not used, no logging will occur.
Report bugs to: villaa-at-gmail-dot-com
realizeCascades (nrCascadeSim) home page: <https://github.com/villano-lab/nrCascadeSim>
General help nrCascadeSim Software: <https://nrcascadesim.readthedocs.io/en/latest/>
gerudo7@Teldrassil ~/g/nrCascadeSim (bugfix/documentation-issues)>                                           (root_env) 
```

Reply:
@gerudo7 would cut off around the first sentence for "verbose" and "log" inputs; go to new line and align it with the texts starting with "Print" and "Log"

Reply:
Done!